packages/shared/constants.ts

```diff
@@ -35,3 +35,9 @@ export const enum NOTIFICATION_TYPE {
   REACTION,
   WEBXDC_INFO,
 }
+
+export const enum SCAN_CONTEXT_TYPE {
+  DEFAULT = 'DEFAULT', // default context, no restrictions on qr code types
+  OTHER_SERVER = 'OTHER_SERVER', // onboarding with another server than the default server
```

Comment:
The naming is somehow inconsistent, but so is the wording in the UI.
Clicking on the "Use other server" button you find a "Scan Invitation Code" button which can be used to san a DC Account url which is not an invitation code but sets the chatmail server to another URL...